.github/workflows/publish-verify.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish verify plugin
+
+on:
```

Comment:
question - shouldn't we only run publish stuff on main branch (maybe it is a stupid question given I don't have great knowledge on github flowers)?

Reply:
This option is there to give us more flexibility in case we want to publish an internal / beta version that is not yet on the main branch.

Reply:
got it